src/libstd/process.rs

```diff
@@ -253,6 +253,10 @@ impl Command {
     /// Builder methods are provided to change these defaults and
     /// otherwise configure the process.
     ///
+    /// The `PATH` from the Command's environment will be searched
+    /// if the program contains only a single path component, as for
+    /// `execvp`.
```

Comment:
I would just give a couple of examples - "For example, `PATH` will be searched for `rustc` but not `bin/rustc`".